tests/integration/pilot/vm/testdata/OS_Support_List

```diff
@@ -0,0 +1,6 @@
+app_sidecar_trusty
+app_sidecar_xenial
+app_sidecar_focal
+app_sidecar_debian8
+app_sidecar_debian9
+app_sidecar_debian10
```

Comment:
This file should be stored somewhere else, somewhere that is accessible to all tests. Also, format is open to change. JSON with Hub, Tag and Image field could be a potential format.